estate/models/EstateProperty.py

```diff
@@ -0,0 +1,44 @@
+from datetime import datetime
+from odoo import models, fields
```

Comment:
```suggestion
from odoo import fields, models

from datetime import datetime
```
Odoo import should be at top and in alphabetic order